src/room007/models/word_tag_predictor.py

```diff
@@ -15,117 +17,130 @@
 from sklearn.linear_model import LogisticRegression
 
 from room007.models import model
+from room007.data import info
 
+import spacy
+#nlp = spacy.load('en')
```

Comment:
I can take all those spacy things out. Had to do with the original plan to test these spacy features.
Nothing fancy or magic happening here. the nlp = spacy.load() is just a demanding call that is why it happens on module level. I commented it out because I didn't want it to load while I was not yet generating those spacy features.